wire-ios-data-model/Source/MLS/Migration/ProteusToMLSMigrationCoordinator.swift

```diff
@@ -117,6 +127,32 @@ public class ProteusToMLSMigrationCoordinator: ProteusToMLSMigrationCoordinating
         }
     }
 
+    private func finalizeMigrationIfNeeded() async throws {
+        let tuples = try await fetchMixedConversations()
+        if !migrationForceTimeHasArrived { try await syncUsersWithTheBackend() }
+
+        for tuple in tuples {
+            await finalizeConversationMigration(conversation: tuple.conversation)
+        }
+
+    }
+
+    private func syncUsersWithTheBackend() async throws {
+        let fetchRequest = ZMUser.fetchRequest()
+        let result = try context.fetch(fetchRequest) as? [ZMUser]
+        guard let result else { return }
+        let ids = result.compactMap { $0.qualifiedID }
```

Comment:
It's hard to read `result`and `ids`. I would suggest giving more meaningful names.

Reply:
Done!